packages/@react-aria/selection/src/useSelectableItem.ts

```diff
@@ -299,10 +299,16 @@ export function useSelectableItem(options: SelectableItemOptions): SelectableIte
     }
   };
 
+  let defaultPressProps = shouldSelectOnPressUp ? {
+    onMouseDown: pressProps.onMouseDown,
+    onTouchStart: pressProps.onTouchStart,
+    onPointerDown: pressProps.onPointerDown
```

Comment:
I think this has some implications with usePress as they are the listeners being returned by the hook. If we only attach these, then we don't do any usePress cleanup on mouse up for example.

The issue is that we need `longPressEnabledOnPressStart` to be true right? We should be able to do this up in onPress/onPressStart.